lib/Net/DRI/Protocol/EPP/Extensions/GoDaddy.pm

```diff
@@ -0,0 +1,46 @@
+## Domain Registry Interface, GoDaddy New gTLD EPP extensions
+##
+## Copyright (c) 2021 Patrick Mevzek <netdri@dotandco.com>. All rights reserved.
+##           (c) 2021 Michael Holloway <michael@thedarkwinter.com>. All rights reserved.
+##           (c) 2021 Paulo Jorge <paullojorgge@gmail.com>. All rights reserved.
+##
+## This file is part of Net::DRI
+##
+## Net::DRI is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+##
+## See the LICENSE file that comes with this distribution for more details.
+####################################################################################################
+
+package Net::DRI::Protocol::EPP::Extensions::GoDaddy;
+
+use strict;
+use warnings;
+
+use base qw/Net::DRI::Protocol::EPP/;
+
+####################################################################################################
+
+sub setup
+{
+ my ($self,$rp)=@_;
+ $self->ns({neulevel => ['urn:ietf:params:xml:ns:neulevel-1.0','neulevel-1.0.xsd']});
+ return;
+}
+
+sub default_extensions {
+ my ($self,$pp) = @_;
+ $self->{brown_fee_version} = $pp->{brown_fee_version} if exists $pp->{brown_fee_version};
+ my @ext = qw/GracePeriod SecDNS LaunchPhase IDN AllocationToken NeuLevel::CO NeuLevel::Message NeuLevel::EXTContact NeuLevel::WhoisType ARI::KeyValue CentralNic::Fee GoDaddy::MZB/;
```

Comment:
... and also over here you are still including all of these extensions...? I realise that GoDaddy uses these in other platforms, but its probably worth then defining these platforms like we do with [VeriSign platforms](https://github.com/thedarkwinter/Net-DRI/tree/master/lib/Net/DRI/Protocol/EPP/Extensions/VeriSign/Platforms) so then you can have the MZB platform that does not include all of these, and the default GoDaddy (or DNRS or ARI...) for others that does not include MZB extension.